specs/ingestion/common/schemas/source.yml

```
@@ -126,9 +126,30 @@ SourceBigCommerce:
     storeHash:
       type: string
       description: The store hash identifying the store the shopper is signing in to.
+    channels:
+      type: array
+      items:
+        $ref: "#/BigCommerceChannel"
   required:
     - store_hash
 
+BigCommerceChannel:
+  type: object
+  additionalProperties: false
+  properties:
+    id:
+      type: integer
+      description: The ID of the bigcommerce channel.
+    currencies:
+      type: array
+      description: An array of currencies for the given channel `ID`, a currency is a trigram string that represents the currency code.
+      example: ["usd","eur"]
```

weird that the linter didn't catch that
```suggestion
      example: ['usd','eur']
```

---

hmmmmmmmmmmmm true